src/utils/convert-filter.js

```diff
@@ -9,6 +9,12 @@ const convertFilter = (filter) => {
     const { property, value } = filterProperty
     switch (property.type()) {
     case 'string':
+      if (property.sequelizePath.values) {
```

Comment:
This is the core of the change: do not use the `LIKE` operator for `ENUM` types.

Another option for the `if` condition could have been:
```
property.sequelizePath.type instanceof DataTypes.ENUM
```
but I prefer this solution suggested by @bakkerthehacker